I did my homework5 without error. I'll do it later

DenisShip opened the pull request.

DenisShip committed:
I did my homework5 without error. I'll do it later

DenisShip committed:
I did my homework5.

efserg commented on src/main/java/shipilovJava1/homeWork5/Employee.java:
```diff
@@ -0,0 +1,57 @@
+
+    }
+
+    public static void main(String[] args) {
+        printInfoEmployee();
+
+
+
+    }
```
Э, нет, так не пойдет.
Метод main перенесите в отдельный класс.
В классе Employee должен быть метод, выводящий информацию о сотруднике на печать. Этот метод не должен быть статическим

DenisShip replied:
Исправил, но ни как не пойму с методом , вывода информации о обьекте

efserg reviewed (changes requested):
Добрый день.
Пока не принято, жду исправлений

DenisShip committed:
I did my homework5 with you comments / не могу понять как распечатать информацию о обьекте
public void print ()                   System.out.println("name, position, email, phone,salary,age");
